packages/wonder-blocks-core/components/clickable-behavior.js

```diff
@@ -90,6 +90,10 @@ type Props = {|
 |};
 
 type State = {|
+    ...ClickableState,
```

Comment:
As you are not adding more keys here, I would suggest to remove this `State` type and only use `ClickableState` instead.

Reply:
good suggestion, I included it in the latest commit